PatientRegistrationApp/src/components/GreetingPage.vue

```diff
@@ -114,10 +114,10 @@ export default {
     {
 		//add logic to check form contents
 		var valid = true
-		var message = "Woops! You need to enter the following fields:"
+		var message = "Woops! You need to enter the following field:"
 	
 			if (this.preferredLanguage == "") {
-			message += "*Preferred Language"
+			message += "Preferred Language"
```

Comment:
You need to add a space preceeding the `<space here>Preferred Language"` or after the `...field:<space here>'`. This still doesn't look right. 

Can you also add in 1 card pointed at 2 points to adjust all the other error messages for human readable text like we're doing here? 

![image](https://user-images.githubusercontent.com/67079030/97932685-43ed5980-1d36-11eb-99e8-073ed65e3032.png)